include/deal.II/grid/reference_cell.h

```diff
@@ -2787,94 +2794,36 @@ ReferenceCell::get_combined_orientation(
                     "the number of vertices of the cell "
                     "referenced by this object."));
 
-  const auto v0_equals = [&](const std::initializer_list<const T> &list) {
-    Assert(list.size() == n_vertices(), ExcInternalError());
-    return std::equal(vertices_0.begin(), vertices_0.end(), std::begin(list));
+  auto compute_orientation = [&](const auto &table) {
+    for (unsigned char o = 0; o < table.size(); ++o)
+      {
+        bool match = true;
+        for (unsigned int j = 0; j < table[o].size(); ++j)
+          match = match && vertices_0[j] == vertices_1[table[o][j]];
```

Comment:
Would you mind adding some braces,
```suggestion
          match = match && (vertices_0[j] == vertices_1[table[o][j]]);
```